localPackages/site_developer/Configuration/TypoScript/Setup/Frontend/Page.typoscript

```diff
@@ -0,0 +1,42 @@
+page = PAGE
+page {
+    includeCSS {
+        bootstrap = EXT:site_developer/Resources/Public/Css/bootstrap.min.css
+        custom = EXT:site_developer/Resources/Public/Css/custom.css
+    }
+
+    meta {
+        viewport = width=device-width, initial-scale=1
+    }
+
+    10 = FLUIDTEMPLATE
+    10 {
+        templateName = Default
+
+        layoutRootPaths {
+            10 = EXT:site_developer/Resources/Private/Layouts/Page/
```

Comment:
I propose to name the site package `site` - that way we avoid the underscore, and the package name then also makes sense for TCCI and TCCE trainings.

Reply:
Thought to use separate sites and site packages within the same installation.
This way we would prevent one group from changing stuff in other areas.